install-prebuilt-uefi/hikey960-efi-prebuilt.sh

```diff
@@ -35,6 +36,10 @@ RELEASE=debug
 -r|--release)
 RELEASE=release
 ;;
+-t|--tty_device)
+DEVICE=$2
```

Comment:
Please update '-h' part.  See Line 25~30.

Reply:
Thanks for pointing that out! I've updated it now.